### DISCUSSION TITLE
Devices Struktur (v3.2.0) geändert?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2614] [General]
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/20f374d7-e591-4a89-8ccd-eb7c6d95e371)
Was wurde denn an der Devices Struktur geändert? Mir fällt es nicht auf.
Warum ist dort keine Signature?

**Comment by Zefau** [comment]
Die Tags wurden generiert auf Basis der Import-Informationen. Die Import-Informationen (Manufacturer) wurden demnach gelöscht.

**Comment by mcuiobroker** [comment]
Ok. Warum keine Signature wie bei den anderen?

**Comment by Zefau** [comment]
Du kommst wahrscheinlich gerade aus dem Konverter. Da wird keine Signatur gesetzt. Wenn du die Geräte wieder änderst, sollte die wieder erscheinen.

**Comment by mcuiobroker** [comment]
Nach der Änderung wird aber die v3 wieder angezeigt, anstatt v3.2
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/6e5585a2-3558-4765-86e9-27a607d5177f)

**Comment by Zefau** [comment]
Jo, stimmt, beim Speichern wird die falsche Version gesetzt.

**Comment by mcuiobroker** [comment]
Schon korrigiert? Oder soll ich ein Issue aufmachen, damit man es nicht vergisst?
